components/common/Suspense/CustomSuspense.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+import { CustomSuspenseProps } from '../../../lib/types/mainTypes'
+
+export default function CustomSuspense({ fallback, maxDuration, children }: CustomSuspenseProps) {
+  const [loadingDone, setLoadingDone] = useState(false)
```

Comment:
상태명때문에 해당 상태 타입이 boolean임을 유추하기 힘든 것 같아요!
isLoadingDone이 낫지 않을까 싶지만, 이건 개인 취향에 따른 부분이라 반드시 적용하지는 않으셔도 괜찮습니다~
```suggestion
  const [isLoadingDone, setIsLoadingDone] = useState(false)
```

Reply:
is붙이는게 더 명확하겠네요! 수정하겠슴돠